### PR TITLE
cv / portfolio

### PR DESCRIPTION
**Pull request by MuyZan**
please check my css

### CODE REVIEW

File: staff/MuyZan/css-layout-simple-adaptive/styles/main.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    height: 20%;
+  }
+  main {
+    height: 60%;
+  }
+  
+  footer {
+    height: 20%;
+  }
+
```
Comment (agandia9) [inline staff/MuyZan/css-layout-simple-adaptive/styles/main.css]: perfect 🎉

File: staff/MuyZan/css-layout-simple/styles/main.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+a:focus{
+    color: rgb(223, 58, 85);
+    text-decoration: overline;
+    
+}
+
+li{
+}
```
Comment (agandia9) [inline staff/MuyZan/css-layout-simple/styles/main.css]: li without styles!..

Comment (manuelbarzi) [reply staff/MuyZan/css-layout-simple/styles/main.css]: TIP css empty rules are considered a mistake. remove empty rules.

File: staff/MuyZan/css-layout-simple-adaptive/styles/desktop.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
+nav{
+    width: 20%;
+    float: left;
+    height: 100%;
+    display:inline-block;
```
Comment (manuelbarzi) [inline staff/MuyZan/css-layout-simple-adaptive/styles/desktop.css]: TIP is ```display: inline-block```rule required?

File: staff/MuyZan/css-layout-simple-adaptive/styles/desktop.css [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    }
+    
+    section{
+        width: 60%;
+        display:inline-block;
+        height: 100%;
+    } 
+    aside{
+        width: 20%;
```
Comment (manuelbarzi) [inline staff/MuyZan/css-layout-simple-adaptive/styles/desktop.css]: TIP group common rules in one single statement of grouped selectors.

SEE "Grouping Selectors" https://www.w3schools.com/css/css_syntax.asp

**Comment by lgimeneza** [comment]
Fine! +1

File: staff/MuyZan/css-layout-simple-adaptive/styles/main.css (hunk 1 above)
Comment (MuyZan) [reply staff/MuyZan/css-layout-simple-adaptive/styles/main.css]: Thanks!

File: staff/MuyZan/css-layout-simple/styles/main.css (hunk 2 above)
Comment (MuyZan) [reply staff/MuyZan/css-layout-simple/styles/main.css]: Thanks you! Done!!!

File: staff/MuyZan/css-layout-simple-adaptive/styles/desktop.css (hunk 4 above)
Comment (MuyZan) [reply staff/MuyZan/css-layout-simple-adaptive/styles/desktop.css]: Done! Thanks!

File: staff/MuyZan/css-layout-simple-adaptive/styles/desktop.css (hunk 3 above)
Comment (MuyZan) [reply staff/MuyZan/css-layout-simple-adaptive/styles/desktop.css]: The css explodes without it.
I tried to put the `display: inline-block` inside a main tag that includes `<nav><section><aside>` tags and delete this rule in the childs, but all crash.

Any tip, please?

**Comment by MuyZan** [comment]
Curriculum totally updated with flexbox. Please, check it!

**Comment by MuyZan** [comment]
Portfolio updated: now it is responsive (using grid and flexbox), please, check it

**Comment by MuyZan** [comment]
please, check it!

**Review by agandia9 — COMMENTED** [review]
OK!

**Pull request merged**